### DISCUSSION TITLE
Suggested new alias

### DISCUSSION THREAD

**Discussion by kfries** [ohmyzsh/ohmyzsh#10706] [Ideas]
I am not sure how this would look, but I almost always use this same kubectl command on most clusters.  It retrieves the dashboard login secret token from the K8s cluster.  It would be nice to see if there was a way to get it added into the regular plugin so I don’t need to add it every time, and others can benefit also:

Kubernetes Get Dashboard Token:
`alias kgdt=‘kubectl get secret -n kubernetes-dashboard $(kubectl -n kubernetes-dashboard get sa/admin-user -o jsonpath=“{.secrets[0].name}”) -o go-template=“{{.data.token | base64decode}}” | pbcopy; echo Dashboard Token added to clipboard’`

This requires the dashboard set up in the standard way according to Kubernetes directions, with the default access account, so it is not appropriate for all users, but hopefully somebody can do something with this code

**Comment by carlosala** [comment]
Looks like an advanced alias, maybe should be placed in your `zshrc` (and probably as a function, and not an alias). By the way, if many people would be interested, we can think on adding it!

**Comment by rwmitchell** [comment]
I'm assuming based on the use of pbcopy, it is limited to MacOS?
